lib/cli/Cargo.toml

```diff
@@ -188,3 +188,6 @@ bin-dir = "bin/{ bin }"
 [package.metadata.binstall.overrides.x86_64-pc-windows-msvc]
 pkg-url = "{ repo }/releases/download/v{ version }/wasmer-windows-amd64.{ archive-format }"
 bin-dir = "bin/{ bin }.exe"
+
+[package.metadata.docs.rs]
+rustc-args = ["--cfg", "docsrs"]
```

Comment:
Is there some workspace support for package.metadata yet? 

I guess even if there was, it wouldn't work for nested keys, and we have other bininstall related metadata?

Reply:
I know there is a `[workspace.metadata]` table, but I'm not sure if docs.rs's build system checks it. According to [the `cargo` docs](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-metadata-table), it's up to individual tools to use check that, though.

The `[workspace.package]` table only supports the following keys:

  |  
-- | --
authors | categories
description | documentation
edition | exclude
homepage | include
keywords | license
license-file | publish
readme | repository
rust-version | version